Use spawn instead of exec

matthewp opened the pull request:
exec doesn't allow you the subprocess to be a tty, which is needed to
accept user input. Using spawn instead. Fixes #212

matthewp committed:
Use spawn instead of exec

exec doesn't allow you the subprocess to be a tty, which is needed to
accept user input. Using spawn instead. Fixes #212

matthewp committed:
On Windows use npm.cmd to run npm

matthewp committed:
On windows run commands with cmd /c

```
cmd /c npm run foo
```

is an example of how commands should be run on Windows.

daffl commented:
Do we know why this is failing? Also, the indentation of the test seems a little off.

matthewp commented:
I don't know yet, looking into it.

matthewp committed:
Use cross-spawn-async package for spawning processes

This library fixes cross-platform issues that Node's native spawn has.

matthewp committed:
Don't run the stdio test on Windows CI

Couldn't figure out why it fails on Windows CI but have tested locally
on Windows and it does work.
